card/lib/card/format.rb

```diff
@@ -224,7 +224,11 @@ def hide_views args
     end
     
     def parse_view_visibility val
-      (val || '').split( /[\s\,]+/ ).map { |view| canonicalize_view view }
+      if val.kind_of? Array
+        val.map { |view| canonicalize_view view }
+      else
+        (val || '').split( /[\s\,]+/ ).map { |view| canonicalize_view view }
```

Comment:
The method is used for the 'show' and 'hide' inclusion options. An empty entry would be uncommon. Moreover the string handing didn't change. I added support for arrays so that I can call
`nest my_card, :hide=>[ view1, view2, view3]` and don't have to generate a string first or convert the array to a hash  `{view1=>:hide, view2=>:hide, view3=>:hide}`